packaging/nnstreamer.spec

```diff
@@ -264,6 +285,14 @@ popd
 %defattr(-,root,root,-)
 %{_prefix}/lib/nnstreamer/filters/libnnstreamer_filter_tensorflow-lite.so
 
+%files python-filter
+%defattr(-,root,root,-)
+%{_prefix}/lib/nnstreamer/filters/libnnstreamer_filter_python.so
```

Comment:
How about replacing 'lib' with '${_libdir} for the portability.

Reply:
Got it